parquet-hadoop/src/main/java/parquet/hadoop/MemoryManager.java

```diff
@@ -39,16 +40,19 @@
 public class MemoryManager {
   private static final Log LOG = Log.getLog(MemoryManager.class);
   static final float DEFAULT_MEMORY_POOL_RATIO = 0.95f;
+  static final long DEFAULT_MIN_MEMORY_ALLOCATION = 0;
```

Comment:
We could set it to something like 10MB, which is my best guess (without any empirical evidence).  We give tasks 1.5 GB which only safely allows for about 10 writers per task with the default 128MB Row Group setting.  A 10MB lower bound would probably push us to 100 writers, which would likely account for most situations.

Any other suggestions are welcome.